courses/free-courses-hi.md

```diff
@@ -248,6 +248,7 @@
 
 ### Git and GitHub
 
+* [Complete Git and Github Tutorials for Beginners](https://youtu.be/Ez8F0nW6S-w?si=BLBMUjVWq4lDerbV) - Apna College
```

Comment:
We do not allow shortened URLS. See https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes.